deploy/anc_health_facility_visit_views.psql

```diff
@@ -0,0 +1,167 @@
+-- Deploy afya-tek:anc_health_facility_visit_views to pg
+
+BEGIN;
+
+-- we need to create the following table
+--  public.anc_health_facility_visit
+-- Unfortunately these ^^ tables exist in the database but not in the migrations
```

Comment:
Nitpick but this is one table yet referred to as "these tables"

Reply:
LOL. Fixed.